ide/gsf.testrunner/nbproject/org-netbeans-modules-gsf-testrunner.sig

```diff
@@ -159,6 +159,8 @@ meth public java.awt.im.InputMethodRequests getInputMethodRequests()
 meth public java.awt.image.ColorModel getColorModel()
 meth public java.awt.image.VolatileImage createVolatileImage(int,int)
 meth public java.awt.image.VolatileImage createVolatileImage(int,int,java.awt.ImageCapabilities) throws java.awt.AWTException
+meth public java.awt.peer.ComponentPeer getPeer()
```

Comment:
Actually it would be better to remove all the `java.awt.peer.ComponentPeer getPeer()` occurences (manually). I have a feeling that the method has disappeared in JDK11 and as such, when one runs the sigtest on a new JDK, it fails saying that this method is missing. That is a false alarm.

If possible, please remove all the 
```
meth public java.awt.peer.ComponentPeer getPeer()
 anno 0 java.lang.Deprecated()
```
occurrences before integrating.

Reply:
done :D